osu.Game.Rulesets.Osu.Tests/TestSceneSpinnerRotation.cs

```diff
@@ -194,13 +193,7 @@ public void TestSpinUnaffectedByClockRate(double rate)
 
             addSeekStep(0);
 
-            AddStep("adjust track rate", () => MusicController.CurrentTrack.AddAdjustment(AdjustableProperty.Tempo, new BindableDouble(rate)));
-            // autoplay replay frames use track time;
-            // if a spin takes 1000ms in track time and we're playing with a 2x rate adjustment, the spin will take 500ms of *real* time.
-            // therefore we need to apply the rate adjustment to the replay itself to change from track time to real time,
-            // as real time is what we care about for spinners
-            // (so we're making the spin take 1000ms in real time *always*, regardless of the track clock's rate).
-            transformReplay(replay => applyRateAdjustment(replay, rate));
+            AddStep("adjust track rate", () => Player.GameplayClockContainer.UserPlaybackRate.Value = rate);
```

Comment:
Could you explain where I went wrong with this before? With this change it feels like the test is testing something else entirely (I meant to simulate DT/HT without actually applying DT/HT as it's more involved).

Reply:
i find the comment here very hard to parse, will need to take another look at it.

Reply:
To be fair now that I think about it, I'm pretty sure it's a hack to bypass the fact that autoplay doesn't generate spinner frames correctly when rate mods are applied. But all of this rate-affecting stuff is super confusing, so I might well be talking nonsense.

Reply:
Has this been addressed to not be required?

Reply:
I suggest you read through this and make your own judgement. As far as I can tell it was not doing what we want.

Reply:
Yeah I think it is fine to go without. Spinner autoplay is broken right now with rate affecting mods and should be fixed properly.